clouddriver-web/src/main/groovy/com/netflix/spinnaker/config/LoggingHandlerExceptionResolver.groovy

```diff
@@ -14,22 +14,25 @@
  * limitations under the License.
  */
 
-package com.netflix.spinnaker.clouddriver.controllers
+package com.netflix.spinnaker.config
 
 import groovy.util.logging.Slf4j
-import org.apache.commons.lang.exception.ExceptionUtils
-import org.springframework.web.bind.annotation.ControllerAdvice
-import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.core.Ordered
+import org.springframework.core.annotation.Order
+import org.springframework.stereotype.Component
+import org.springframework.web.servlet.ModelAndView
+import org.springframework.web.servlet.handler.AbstractHandlerExceptionResolver
 
 import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
 
 @Slf4j
-@ControllerAdvice
-class LoggingExceptionHandler {
-
-  @ExceptionHandler(value = Exception.class)
-  def defaultErrorHandler(HttpServletRequest request, Exception e) {
-    log.error("Error occurred handling request for ${request.requestURL}: ${ExceptionUtils.getFullStackTrace(e)}")
-    throw e
```

Comment:
It believe it was to help address this part in https://github.com/spinnaker/spinnaker/issues/657:

> Furthermore, Retrofit errors (such as 404s or 500s) are not printed to the logs, which would help in debugging issues.

I believe I meant RestController instead of retrofit.